tests/fx/converters/test_linalg_ops_fx.py

```diff
@@ -1,8 +1,27 @@
 import pytest
 import torch
-from fx_test_utils import convert_to_mgx, verify_outputs
+from fx_test_utils import FuncModule, convert_to_mgx, verify_outputs
 
 
 @pytest.mark.skip(reason="linalg.norm converter not implemented")
 def test_linalg_norm():
     pass
+
+
+@pytest.mark.parametrize('ord, dim, keepdim', [
+    (0, None, False),
+    (1, None, True),
+    (torch.inf, 0, False),
+    (-torch.inf, -1, True),
+    (2, 1, True),
```

Comment:
```suggestion
    (2, 1, True),
    (None, 1, True),
```
I don't think your code will handle that, but it would be only 1 or 2 lines to add it.

Reply:
It should default to 2.

Reply:
See new test case, I think thats what youre really asking for. Pytorch defaults the value of `ord` to 2 when not given, so the converter still works as expected

Reply:
The display of changes by github is getting a bit messy, but what I see is a test case in `fx` where the inputs are `()` and no corresponding case for `dynamo`.  I'm going to assume that's what you intended and call it complete.

Reply:
Sorry, missed this comment before merging, hope it makes sense though